backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -737,6 +733,41 @@ class SchedulingService implements ISchedulingService {
     }
   }
 
+  async updateSchedulingByRecurringDonationId(
+    recurringDonationId: string,
+    scheduling: UpdateSchedulingDTO,
+  ): Promise<void> {
+    try {
+      const updatesSnakeCase: Record<string, unknown> = {};
+      const startDateTime = new Date(scheduling.startTime!);
```

Comment:
just curious, why the non-null assertion here for startTime?

Reply:
it was throwing an error if i dont have that since type undefined is not assignable to new Date()